tests/testsuite/jobserver.rs

```diff
@@ -57,6 +57,8 @@ fn validate(_: &str) {
 fn make_exe() -> &'static str {
     if cfg!(windows) {
         "mingw32-make"
+    } else if cfg!(target_os = "aix") {
```

Comment:
> Is GNU Make always installed as gmake on IBM AIX (I know the g prefix is a convention though)?

Yes. It's installed via https://www.ibm.com/support/pages/aix-toolbox-open-source-software-downloads-alpha and there will be symbol link named `/usr/bin/gmake` which links to `/opt/freeware/bin/make`.

> Would it be more preferable that downstream exclude these tests when running the test suite?

Since adding CI for AIX on github is hard, but we also want to deliver the same quality as tier-1 targets, we are maintaining an internal CI on AIX, running the almost the same CI tasks(testing, packaging...) as `rust-lang` org does. So for our strategy, we try our best to upstream what we are able to, while not to break community's CI.

Reply:
Sounds great. Thanks for your reply!